config/locales/it.yml

```diff
@@ -578,8 +579,46 @@ it:
+         </p> 
+         <p>
+         Per questo abbiamo sentito il bisogno di imitare esperienze che all’estero hanno creato comunità di ottima qualità per contenuti e dibattito. Seguendo l’esempio di <a href=\"https://journalduhacker.net\" target=\"_black\">journalduhacker</a> che ha portato in lingua francese il format originariamente creato da <a href=\"https://lobste.rs\" target=\"_black\">lobste.rs</a>, abbiamo deciso di tradurre in lingua italiana il loro software e su questo costruire gambe.ro.
+          </p>
+          <h3>Obiettivi della comunità</h3>
+          <p>
+          gambe.ro è sia un aggregatore di notizie che uno spazio di discussione. Gli argomenti che vuole coprire sono le pratiche dello sviluppo software e tutto ciò che le circonda, inclusi i cambiamenti politici, sociali ed economici che il software porta nella nostra società. 
+          </p>
+          <p>
+          gambe.ro vuole essere anche uno spazio di crescita personale e professionale per persone di tutti i gradi di competenza. Perciò sono benvenute discussioni e riflessioni su percorsi di studio, carriera, eventi e conferenze se collegati ai temi di interesse della comunità.
+          <p>
+          gambe.ro non vuole essere uno spazio di supporto tecnico né per sviluppatori né per utilizzatori. Non vuole essere neppure uno spazio di discussione della tecnologia in senso esteso: vi invitiamo a trattare temi come hardware, telefonia o futurologia nei numerosi spazi di discussione già esistenti.
+             </p>
+              
+              <hr3>Moderazione</h3>
+              <p>
+              Il team di moderazione si riserva di intervenire in caso vengano pubblicati sulla piattaforma contenuti illegali o esplicitamente fuori contesto rispetto ai temi di interesse della comunità. Confidiamo nella capacità della comunità di auto-moderarsi attraverso lo strumento del downvote che invitiamo ad usare con giudizio. La moderazione interverrà inoltre per reprimere comportamenti tossici e ostili verso membri della comunità o terzi, in linea con i principi e le regole esposte nel Code of Conduct.
+              </p>
+              
+              <h4>Downvote</h4>
+              <p>
+              Lo strumento del downvote su gambe.ro è da intendersi come un’espressione di dissenso verso la presenza del commento che si downvota. Non deve rispecchiare invece il vostro parere personale sull’opinione espressa dal commento. Per questo motivo ogni downvote è accompagnato da una motivazione. Se nessuna delle motivazioni presenti si applica al downvote che volete assegnare, probabilmente il contenuto non si merita un downvote.
```

Comment:
Troviamo un analogo italiano a _downvote_ o vogliamo adoperare il solito gergo delle altre piattaforme anglofone?

Reply:
qualunque traduzione sarebbe forzata visto che nessuno l'ha mai tradotto in una piattaforma italiana.